module/behaviour/planning/FixedWalk/src/FixedWalk.cpp

```diff
@@ -19,97 +19,108 @@
-                NUClear::log("!!!!!!!!!!!!!!!!!!!!WARNING: Walk finished prematurely!!!!!!!!!!!!!!!!!!!!");
-            }
-        });
-
-		on<Trigger<FixedWalkCommand>, With<Sensors>, Sync<FixedWalk>>().then([this](const FixedWalkCommand& command, const Sensors& sensors){
-            if(!active && !command.segments.empty()){
-                active = true;
-    			segmentStart = NUClear::clock::now();
-    			beginningOrientation = sensors.world.rotation();
-    		}
-    		for(auto& segment: command.segments){
-    			walkSegments.push_back(segment);
-    		}
-            log("FixedWalk::FixedWalkCommand - Total walk segments pushed back:", walkSegments.size(), command.segments.size());
-		});
-    }
-
-    std::unique_ptr<WalkCommand> FixedWalk::getWalkCommand(const FixedWalkCommand::WalkSegment& segment, NUClear::clock::duration t, const Sensors&){
-     	double timeSeconds = std::chrono::duration_cast<std::chrono::seconds>(t).count();
-    	arma::vec2 directionInOriginalCoords = (segment.curvePeriod != 0 ? Rotation2D::createRotation(2 * M_PI * timeSeconds / segment.curvePeriod) : arma::eye(2,2) ) * segment.direction;
-    	arma::vec2 direction =  arma::normalise(directionInOriginalCoords);
-    	auto result = std::make_unique<WalkCommand>();
-        result->command.xy() = segment.normalisedVelocity * direction;
-    	result->command.angle() = segment.normalisedAngularVelocity;
-    	return std::move(result);
-    }
-
-
-}  // skills
-}  // behaviours
-}  // modules
+            });
+
+            on<Trigger<FixedWalkCommand>, With<Sensors>, Sync<FixedWalk>>().then(
+                [this](const FixedWalkCommand& command, const Sensors& sensors) {
+                    if (!active && !command.segments.empty()) {
+                        active               = true;
+                        segmentStart         = NUClear::clock::now();
+                        beginningOrientation = sensors.world.rotation();
+                    }
+                    for (auto& segment : command.segments) {
+                        walkSegments.push_back(segment);
+                    }
+                    log("FixedWalk::FixedWalkCommand - Total walk segments pushed back:",
+                        walkSegments.size(),
+                        command.segments.size());
+                });
+        }
+
+        std::unique_ptr<WalkCommand> FixedWalk::getWalkCommand(const FixedWalkCommand::WalkSegment& segment,
+                                                               NUClear::clock::duration t,
+                                                               const Sensors&) {
+            double timeSeconds = std::chrono::duration_cast<std::chrono::seconds>(t).count();
+            arma::vec2 directionInOriginalCoords =
+                (segment.curvePeriod != 0 ? Rotation2D::createRotation(2 * M_PI * timeSeconds / segment.curvePeriod)
+                                          : arma::eye(2, 2))
+                * segment.direction;
```

Comment:
nah the `* segment.direction;` is on a completely new line, even tho it can definitely fit on the line above. Tho, I am not sure there is really a setting that would fix this, that wouldn't really effect other code.